finalPrep/modZeroHero.js

```diff
@@ -55,30 +73,56 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
+var scaryMonster = {
+  name: 'Bingus',
+  smell: 'stinky',
+  weight: 5000,
+  citiesDestroyed: ['Denver','Thornton','Pueblo'],
+  luckyNumbers: [5,7,85],
+  address: [42, 'ronald', 'CO', 82580]
 
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
-
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name,superpower,age){
+  this.name = name;
+  this.power = superpower;
+  this.age = age;
+  this.archNemesis = 'KennyS';
+  this.powerLevel = 100;
+  this.energyLevel = 50;
+}
+sayName(sn){
```

Comment:
What is the purpose of `sn` here as a function parameter?

Reply:
From practice and a tad bit of external help, I was told to always have fluid code. During the majority of my mod 1 prework I was hard coding everything which in an assignment is okay. The parameter is there to make sure if I would like to code anything else when I log or return the input it can be changed within a second, instead of changing the hardcode everytime. If this is bad practice please let me know!

Reply:
Okay understood, I think in this case since the instructions don't specify passing in a function parameter, it's fine to leave it out.